independent-projects/tools/codestarts/src/main/java/io/quarkus/devtools/codestarts/core/CodestartProjectGeneration.java

```diff
@@ -47,11 +48,19 @@ static void generateProject(final CodestartProjectDefinition projectDefinition,
             processor.process(codestart);
         }
         processor.writeFiles();
+
         log.info("\napplying codestarts...");
-        log.info(projectDefinition.getCodestarts().stream()
-                .map(c -> c.getType().getIcon() + " "
-                        + c.getName())
-                .collect(Collectors.joining("\n")));
+        if (!OS.WINDOWS.isCurrent()) {
+            log.info(projectDefinition.getCodestarts().stream()
+                    .map(c -> c.getType().getIcon() + " "
+                            + c.getName())
+                    .collect(Collectors.joining("\n")));
+        } else {
+            log.info(projectDefinition.getCodestarts().stream()
+                    .map(c -> c.getType().getAlternativeIcon() + " " + c.getName())
+                    .collect(Collectors.joining("\n")));
```

Comment:
Shouldn't this be `\r\n`? I wonder if we could use `System.lineSeparator()` instead in both statements

Reply:
Done